css/input.css

```diff
@@ -0,0 +1,124 @@
+
+
+.login_bar {
+	width: 300px;
+	height:45px;
+	background-color:#6B90D9;
+	border-radius: 10px;
+	position: relative;
+    z-index: 1;
+	overflow: hidden;
+	
+	
+}
+
+.login {
+	background-color:#6B90D9;
+	color:#fff;
+	position: absolute;
+	height: 103%;
+	width: 202px;
+	padding-left: 10px;
+	text-align: left;
+	line-height: 50px;
+	vertical-align: middle;
+	-webkit-transition: transform 0.2s ease-in-out 0s;
+	   -moz-transition: transform 0.2s ease-in-out 0s;
+	     -o-transition: transform 0.2s ease-in-out 0s;
+	        transition: transform 0.2s ease-in-out 0s;
+}
+
+.login_active {
+	-webkit-transform: translateX(300px);
+	   -moz-transform: translateX(300px);
+	    -ms-transform: translateX(300px);
+	     -o-transform: translateX(300px);
+	        transform: translateX(300px);
+}
+
+.subscribe_button:hover{
+	background-color: #434ACF;
+	cursor:pointer;
+	color:black;
+}
+
+.subscribe_button {
+	background:#434ACF;
+	position: absolute;
+	top: -50px;
+	left: 200px;

```

Comment:
**right:0** instead of **left: 200px** since you want it to stay to the right, easier to controll, does the same thing